### PR TITLE
chore: package/statement: Add function prepareExtrinsic() to return extrinsic and change cord version to latest

### LINKED ISSUE #180: Move `signAndSubmitTx` out of the function `dispatchRegisterToChain` & `dispatchRevokeToChain`
**Issue:**
The `authorizeBatch()` function currently accepts an array of Extrinsic objects as its argument. However, when utilizing `dispatchRegisterToChain()`, it submits a single transaction directly to the chain.

**Requirement**:
For bulk uploads or revocations of records in the studio.

**Solution**:
To address this, consider removing the `signAndSubmitTx()` function from within `dispatchRegisterToChain()` and instead have it return only the extrinsic. The same has to be done for `dispatchRevokeToChain()`

### PR DESCRIPTION
**Pull request by Vikastc**
fixes: #180

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,6 @@
   "resolutions": {
     "typescript": "^5.3.3"
   },
-  "version": "0.9.3-1rc4",
+  "version": "0.9.3-1r6",
```
Comment (amarts) [inline package.json]: technically, this is not needed now, while making the release we can change it.

**Review by amarts — COMMENTED** [review]
LGTM

package.json and yarn.lock changes can come in as a separate PR IMO

File: package.json (hunk 1 above)
Comment (Vikastc) [reply package.json]: Okay

**Review by amarts — APPROVED** [review]

**Comment by amarts** [comment]
will make a release tomorrow or day after, once we get #184 merged.

**Pull request merged**